spec/controllers/checkout_controller_spec.rb

```diff
@@ -85,6 +85,42 @@
       controller.send(:clear_ship_address)
     end
 
+    context "#update with shipping_method_id" do
+      let(:test_shipping_method_id) { "111" }
+
+      before do
+        # stub order and resetorderservice
+        allow(ResetOrderService).to receive(:new).with(controller, order) { reset_order_service }
+        allow(reset_order_service).to receive(:call)
+        allow(order).to receive(:update_attributes).and_return true
+        controller.stub(:current_order).and_return(order)
```

Comment:
please, use `allow` instead. There has to be a way to check this automatically :thinking:

Reply:
:+1:
done